orga/app/components/auth/login-form.js

```diff
@@ -48,12 +47,20 @@ export default class LoginForm extends Component {
           this.args.organizationInvitationCode,
           email
         );
-      } catch (responseError) {
-        responseError.errors.forEach((error) => {
-          if (error.status === '412') {
-            return this._authenticate(password, email);
-          }
-        });
+      } catch (err) {
```

Comment:
Question : on a commencé à mettre du responseError, pourquoi revenir à 'err' ?

Reply:
`response` et `responseError` sont des noms de variables plutôt à réserver aux appels réseau.

Or la fonction `_acceptOrganizationInvitation` fait un appel à `store.createRecord` qui n'est pas systématiquement un appel réseau puisque c'est du Ember qui fait astucieusement soit des appels réseau soit des appels locaux.

Je n'ai pas voulu appeler cette erreur `emberError`, car quand il n'y a plus de connexion réseau ce n'est pas une erreur Ember qui est lancé.

Reply:
'car quand il n'y a plus de connexion réseau ce n'est pas une erreur Ember qui est lancée.`
Ah bon ? Je crois que j'ai toujours pas bien compris alors...

Reply:
"response et responseError sont des noms de variables plutôt à réserver aux appels réseau"
Encore une question : c'est une norme qui vient d'où ? 
Dans le code Pix, il ne me semble pas avoir vu cette distinction

Reply:
Utiliser `response` et `responseError` pour des noms de variables plutôt réservées aux appels réseau est une pratique que j'ai noté en général sur les différents projets libres dont j'ai lu les sources. Je pense que le code de pix fait cette utilisation également, vu avec un `git grep` sur `mon-pix`, mais effectivement je n'ai aucun ADR ou commentaire ou documentation de Pix qui formalise la chose.